slasher/db/indexed_attestations.go

```diff
@@ -25,8 +25,8 @@ func unmarshalIdxAtt(enc []byte) (*ethpb.IndexedAttestation, error) {
 	return protoIdxAtt, nil
 }
 
-func unmarshalValIDsToIdxAttList(enc []byte) (*slashpb.ValidatorIDToIdxAttList, error) {
-	protoIdxAtt := &slashpb.ValidatorIDToIdxAttList{}
+func unmarshalCompressedIdxAttList(enc []byte) (*slashpb.CompressedIdxAttList, error) {
```

Comment:
CompressedIdx sounds too vague imo, is it validator index? attestation index? block index?... etc

Reply:
It is an indexed attestation, but the `AttestationData` is changed to its `[]byte` attestation data root. That's why I refer to it as "compressed".

Reply:
Makes sense 👍